testcases/misc_tests/p11sak_test.sh

```diff
@@ -70,6 +70,22 @@ echo "** Setting SLOT=30 to the Softtoken unless otherwise set - 'p11sak_test.sh
 
 SLOT=${SLOT:-30}
 
+# check if p11sak is available in the current $PATH - if it isn't, ask for
```

Comment:
I like this! 

Please check if the commit description of this commit is still correct. It still mentiones /usr/local/sbin, etc.